service/ciphering/aesNotRandomEncoder.go

```diff
@@ -0,0 +1,56 @@
+package ciphering
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/hex"
+	"errors"
+)
+
+type aesNotRandomEncoder struct {
```

Comment:
Try to not use negations in naming. Something like `regularEncoder` will suite it well (for now).